zlp-scheduler/features/step_definitions/authentication_steps.rb

```diff
@@ -0,0 +1,66 @@
+Before do
+  # The current sessions_controller requires at least one active term
+  @term = FactoryBot.create(:term, :active=>true)
+  @term.save()
+end
```

Comment:
I just want to make sure I'm understanding what this does correctly. Before anything test-wise is done, it creates an active term in the db? Is that correct?

Reply:
Yes, you are correct. The "before" block will be run before every scenario.